File: User/src/main/java/com/example/microservice/entitiy/Booking.java
package com.example.microservice.entitiy;

import java.time.LocalTime;
import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Booking {

	private Long bookingId;
	private String userId;
	private Long carId;
	private Double amount;
	private Car car;
	private Date pickUpDate;
	private Date dropOffDate;
	private String pickUpLocation;
	private String dropOffLocation;
	private LocalTime pickUpTime;
    private LocalTime dropOffTime;
	private Boolean isBlocked = false;


}

File: Frontend/microservice/src/app/model/booking.ts
import { Time } from "@angular/common";

export class Booking {
    constructor(
        public bookingId: number = 0,
        public userId: string = '',
        public carId: number = 0,
        public amount: number = 0,
        public pickUpDate:Date,
        public dropOffDate:Date,
        public pickUpLocation:string='',
        public dropOffLocation:string='',
        public  pickUpTime:Time,
	    public  dropOffTime:Time,
        public status: string ="",
        public isBlocked:boolean=false

        
    ){}
}
